.mobsf

```diff
@@ -0,0 +1,18 @@
+---
+  ignore-paths:
+  - __MACOSX
+  - skip_dir
+
+  ignore-rules:
```

Comment:
What is the reason for these ignored rules?

Reply:
According to the doc they evaluate these rules differently. When they do not found any pattern in the entire code then triggered these rules. Please check this doc for details https://github.com/MobSF/mobsfscan/blob/main/mobsfscan/rules/patterns/android/kotlin/best_practices.yaml